sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -71,3 +159,65 @@ def teardown(
         raise NotImplementedError(
             "to be implemented in https://jira.expedia.biz/browse/EAPC-7974"
         )
+
+    def _convert_featureview_schema_to_milvus_readable(
+        self, feast_schema: List[Field]
+    ) -> CollectionSchema:
+        """
+        Converting a schema understood by Feast to a schema that is readable by Milvus so that it
+        can be used when a collection is created in Milvus.
+
+        Parameters:
+        feast_schema (List[Field]): Schema stored in VectorFeatureView.
+
+        Returns:
+        (CollectionSchema): Schema readable by Milvus.
+
+        """
+        boolean_mapping_from_string = {"True": True, "False": False}
+        field_list = []
+
+        for field in feast_schema:
+            data_type = self._feast_to_milvus_data_type(field.dtype)
+            field_name = field.name
+            description = field.tags.get("description")
+            is_primary = boolean_mapping_from_string.get(field.tags.get("is_primary"))
+            dimension = field.tags.get("dimension")
```

Comment:
dimensions have to be retrieved from `VectorFeatureView`. Also, the name of the column holding the vector is defined separately in `VectorFeatureView` ("vector_field")